src/entity/link.rs

```diff
@@ -30,7 +30,7 @@ pub trait Linked {
 
             select.query().join_as(
                 JoinType::InnerJoin,
-                unpack_table_ref(&rel.from_tbl),
+                rel.from_tbl,
```

Comment:
Can I caught some other bug as well. This should pass the `TableRef` in directly, so that schema name will also be included if needed.